css/styles.css

```diff
@@ -0,0 +1,85 @@
+html {
+  margin: 0;
+  padding: 0;
+}
+
+body, p {
+  margin: 0;
+  padding: 0;
+  font-family: -apple-system, BlinkMacSystemFont, "helvetica neue", helvetica, arial, sans-serif;
+  font-size: 18px;
+  line-height: 1.4em;
+  letter-spacing: .1em;
+  background: white;
+  text-transform: lowercase;
+}
+
+.homebox {
+  padding-top: 2em;
+  padding-left: 2em;
+  width: 50%;
+}
+
+.textbox {
+  padding-top: 1em;
+  margin: 0 auto;
+  height: auto;
+  width: 50%;
+}
+
+.left {
+  padding: 1em;
+  display: block;
+  text-align: left;
+}
+
+.right {
+  padding: 1em;
+  display: block;
+  text-align: right;
+}
+
+.leftcredit {
+  padding: 1em;
+  position: fixed;
+  bottom: 1em;
+  left: 1em;
+  display: inline;
+  text-align: left;
+}
+
+.rightcredit{
+  padding: 1em;
+  position: fixed;
+  bottom: 1em;
+  right: 1em;
+  display: inline;
+  text-align: right;
+}
+
+/* unvisited link */
+a:link {
+  color: black;
+  text-decoration: none;
+}
+
+/* visited link */
+a:visited {
+  color: black;
+}
+
+/* mouse over link */
+a:hover {
+  color: blue;
+}
+
+/* selected link */
+a:active {
+  color: blue;
+}
+
+@media screen and (max-width: 700px) {
+  .box {
```

Comment:
what's this class that's only selected on like tablet and below screen sizes?

Reply:
ah its messed up, should be for .textbox not .box, ill update. but basically this makes it so on desktop the container is a certain size, centered and on mobile it takes up the entire screen